tests/cockpit-tests

```diff
@@ -63,7 +63,8 @@ function task() {
 }
 
 # Perform N tasks or waits, then restart
-for i in $(seq 1 30); do
+TASKS=300
```

Comment:
This also seems unrelated. This PR keeps the cache volume, so there's no precious state in these containers. It seems useful to not accumulate too much cruft in them, and update to the latest cockpit/tests tag relatively often. Was there a particular reason to change this?

Reply:
Yes, it's related.

In addition to efficiency, we use the host caching to prevent constantly re-downloading images. However in the ```restricted``` case we cannot use that host caching. So the interval between downloading *all* the images again starts to be annoying. 

Hence the bumping up of this number, and I believe it's a reasonable tradeoff between the two factors: 
 * Avoiding cruft
 * Downloading everything constantly

Reply:
I think this is just a follow-up from the discussion above, so let's ignore that for now. (But to reiterate: pods not sharing volumes sounds like a really bad idea IMHO)

Reply:
I don't see why they would need to share volumes. The various pods should be organic, distributed and able to operate on their own. The fact that they do share volumes was purely an optimization for density.

Reply:
IMHO not having ten copies of our 54 GB (!) cache directory on one and the same node is not an "optimziation" - it would be a giant waste to not do it. Also note that this would make pods prone to multiple parallel downloads, which we've just recently [seen to fall over horribly](https://github.com/cockpit-project/cockpit/pull/8740#issuecomment-369581186). This got [fixed](https://github.com/cockpit-project/cockpit/pull/8741), but not sharing a cache directory would bring back this problem.

So I'd ask it the other way around: why would we *nod* use a volume for the cache? That doesn't preclude them from being organic or independent in any way.